### PR TITLE
Fix the build script

### PR DESCRIPTION
**Pull request by pipo02mix**

### CODE REVIEW

File: tools/fake-flux-build [hunk 1]
```diff
@@ -92,7 +92,6 @@ build() {
         cat <<EOF > ${match}/${wcluster}/${subpath}/kustomization.yaml
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-buildMetadata: [originAnnotations]
```
Comment (pipo02mix) [inline tools/fake-flux-build:95]: It does not work for me (I have tried two different flux versions)
```
~/go/src/github.com/giantswarm/workload-clusters-fleet » fake-flux-build build gamma multi-project fwest /  
Error: json: unknown field "buildMetadata"
--------
 » flux version                                                                                            
1.25.2
```

Comment (pipo02mix) [reply tools/fake-flux-build:95]: Without it works fine (I guess annotations are not carried accross

Comment (piontec) [reply tools/fake-flux-build:95]: It's not flux'es property, but kustomize's - test wtih it and should work fine

**Review by piontec — COMMENTED** [review]
The origin thing is not critical in this case, but it's definitely working for us. What system and tool versions do you use?

File: tools/fake-flux-build (hunk 1 above)
Comment (pipo02mix) [reply tools/fake-flux-build:95]: but I have already kustomize and it is not working

Comment (pipo02mix) [reply tools/fake-flux-build:95]: I am running latest version
```
 » fake-flux-build build gamma multi-project fwest /  
Error: json: unknown field "buildMetadata"
---------------------------------------------------------------------------------------------------
» kustomize version                                                                                       
{Version:kustomize/v4.5.5 GitCommit:daa3e5e2c2d3a4b8c94021a7384bfb06734bcd26 BuildDate:2022-05-20T20:21:22Z GoOs:darwin GoArch:amd64}
````

Comment (piontec) [reply tools/fake-flux-build:95]: Definitely works for me....
```
kustomize version
{Version:kustomize/v3.5.4 GitCommit:3af514fa9f85430f0c1557c4a0291e62112ab026 BuildDate:2020-01-11T03:12:59Z GoOs:linux GoArch:amd64}
```
It seems they removed it in the newer versions? Can you try to hide the `kustomize` binary and use the one included in `kubectl`?

Comment (pipo02mix) [reply tools/fake-flux-build:95]: I dont know how to do that? did not find in docs either
And what kubectl version do you use?

I would like to add to the tools docs

Comment (ljakimczuk) [reply tools/fake-flux-build:95]: @pipo02mix this script uses `kubectl kustomize` to generate the manifests. Can you try to remove `kubectl` from here https://github.com/giantswarm/gitops-template/blob/9d3d66f83e4d390a6f0191c4c67638ee2420f0b8/tools/fake-flux-build#L107 and use just the `kustomize` binary and tell me if that works for you?

Comment (pipo02mix) [reply tools/fake-flux-build:95]: using -> `kustomize build ${match}/${wcluster}/${subpath} | envsubst ` works!

IMO it is better to use standalone tool and mark the necessary version in the readme, wdyt?

**Comment by piontec** [comment]
Hey, tested again with 4.5.x series, everything works as expected. Can you please check if you still have this problem?

**Comment by pipo02mix** [comment]
yeah it works now 👋

**Pull request closed without merging**